src/renderer/components/ReaderApp.tsx

```diff
@@ -151,10 +151,10 @@ const queryParams = getURLQueryParams();
 
 // tslint:disable-next-line:no-string-literal
 const publicationJsonUrl = queryParams["pub"];
-const publicationJsonUrl_ = publicationJsonUrl.startsWith(READIUM2_ELECTRON_HTTP_PROTOCOL) ?
+const newPublicationJsonUrl = publicationJsonUrl.startsWith(READIUM2_ELECTRON_HTTP_PROTOCOL) ?
     convertCustomSchemeToHttpUrl(publicationJsonUrl) : publicationJsonUrl;
 
-const pathBase64 = publicationJsonUrl_.replace(/.*\/pub\/(.*)\/manifest.json/, "$1");
+const pathBase64 = newPublicationJsonUrl.replace(/.*\/pub\/(.*)\/manifest.json/, "$1");
```

Comment:
...and just to be clear: this is also incorrect :)  (please add `_` again, or change the variable name if this is confusing)

Reply:
_ is forbidden by linter